lib/up-storage.js

```diff
@@ -353,17 +354,28 @@ Storage.prototype.search = function(startkey, options) {
     },
   });
 
+  let parsePackage = (pkg) => {
+    if (Utils.is_object(pkg)) {
+      stream.emit('data', pkg);
+    }
+  };
+
   req.on('response', (res) => {
     if (!String(res.statusCode).match(/^2\d\d$/)) {
       return stream.emit('error', Error('bad status code ' + res.statusCode + ' from uplink'));
     }
 
-    res.pipe(JSONStream.parse('*')).on('data', (pkg) => {
-      if (Utils.is_object(pkg)) {
-        stream.emit('data', pkg);
-      }
-    });
-    res.on('end', () => {
+
+    // See https://github.com/request/request#requestoptions-callback
+    // Request library will not decode gzip stream.
```

Comment:
You really reply me at 22 hours ago? i have receive notification email at 1 hours ago